mmv1/third_party/tgc/tests/data/example_vertex_ai_dataset.json

```diff
@@ -0,0 +1,21 @@
+[
+   {
+      "name":"//us-central1-aiplatform.googleapis.com/placeholder-WfOZVAEN",
+      "asset_type":"{{region}}-aiplatform.googleapis.com/Dataset",
+      "resource":{
+         "version":"v1beta1",
+         "discovery_document_uri":"https://www.googleapis.com/discovery/v1/apis/{{region}}-aiplatform/v1beta1/rest",
```

Comment:
```suggestion
         "discovery_document_uri":"https://www.googleapis.com/discovery/v1/apis/us-central1-aiplatform/v1beta1/rest",
```

Reply:
Even if I change the value another error will happen, I'm waiting for the responsible analyst's decision to resolve the problem

Reply:
I've already tried this procedure and it doesn't work, in my opinion we should configure magic_modules to not generate the vertex_ai_dataset.go file and then create it as a handwritten case with the necessary changes